client/app/views/steps/password.coffee

```diff
@@ -1,14 +1,24 @@
 StepView = require '../step'
+timezones = require '../../lib/timezones'
+
 
 module.exports = class PasswordView extends StepView
+
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click button': 'doSubmit'
+
+
+    getDataFromDOM: ->
+        # TODO: get user info
+        # from server
+        return {
+            password: @$('input[name=password]').val()
+            onboardedSteps: ['welcome', 'agreement', 'password']
```

Comment:
@gregorylegarec I added this because @CPatchane asked for; maybe we need to be consistent/agree about this; what should I do?